src/lib/npmCommand.ts

```diff
@@ -6,9 +6,12 @@
  */
 /* eslint-disable @typescript-eslint/no-unused-vars */
 
+import { type as osType } from 'os';
```

Comment:
@maggiben why the alias. This confuses old people like me :) I would rather see `os.type()`

Reply:
@peternhale that was me. 
Because if you want to do `os.type()` then you have to bring in _all_ of `os` with `import * as os from 'os';`.
So you can do `import { type } from 'os';` but then calling `type()` isn't very descriptive either, so I aliased it to `osType()`;